### PR TITLE
add support for Harmony

### PR DESCRIPTION
**Pull request by jhd2best**
* add Harmony mainnet and testnet info
* add Harmony multicall smc address

### CODE REVIEW

File: src/networks.json [hunk 1]
```diff
@@ -273,5 +273,25 @@
...
     "ws": ["wss://ws.rupx.io"],
     "explorer": "http://scan.rupx.io"  
+  },
+  "1666600000": {
+    "key": "1666600000",
+    "name": "Harmony Mainnet",
+    "shortName": "HarmonyMainnet",
+    "chainId": 1666600000,
+    "network": "mainnet",
+    "rpc": ["https://api.s0.t.hmny.io"],
```
Comment (bonustrack) [inline src/networks.json:283]: Can you confirm than this is an archive node?

**Comment by bonustrack** [comment]
@jhd2best Thanks for your PR! Could you also add an image here: https://github.com/snapshot-labs/snapshot.js/tree/master/src/networks image from Twitter will work well.

File: src/networks.json (hunk 1 above)
Comment (jhd2best) [reply src/networks.json:283]: sure. it's an archive rpc interface

**Comment by jhd2best** [comment]
@bonustrack already added logo in networks dir.

**Review by bonustrack — APPROVED** [review]
Looks good!

**Pull request merged**